app/services/CategorisationService.scala

```diff
@@ -121,7 +121,6 @@ class CategorisationService @Inject() (
     } yield updatedAnswers
   }
 
-  //TODO this will be refactored out in TGP-1600 but solves the immediate problem
```

Comment:
was it refactored in 1600?

Reply:
There is a separate refactoring ticket for this